src/data/agile-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const agileQuiz = [
+  {
+    Question: "What are the three primary roles in Scrum and their key responsibilities?",
+    Answer: "The three primary roles in Scrum are Product Owner, Scrum Master, and Team Member. The Product Owner maximizes ROI, the Scrum Master guides team performance, and Team Members have authority over work methods.",
+    Distractor1: "Benefits of Agile methodologies.",
+    Distractor2: "Difference between Scrum and Kanban.",
+    Distractor3: "History of project management.",
```

Comment:
it is best to come up with distractors that better matched the question and context.
It makes for a better quiz that way

```suggestion
    Distractor1: "CEO, CTO, web master",
    Distractor2: "Scrum Master, CFO, CTO",
    Distractor3: "Product Owner, COO, Scrum Master",
```